common/koa-middleware/withToggles.js

```diff
@@ -33,16 +43,45 @@ const parseCookies = function(req) {
 function withToggles(ctx, next) {
   const cookies = parseCookies(ctx.req);
   const togglesCookies = cookies.filter(cookie =>
-    cookie.key.startsWith('toggle_')
+    cookie.key.startsWith(cookiePrefix)
   );
   const toggles = togglesCookies.reduce((acc, cookie) => {
     return Object.assign({}, acc, {
-      [cookie.key.replace('toggle_', '')]: cookie.value === 'true',
+      [cookie.key.replace(cookiePrefix, '')]: cookie.value === 'true',
     });
   }, {});
 
   ctx.toggles = { ...defaultToggleValues, ...toggles };
+  enableDisableToggle(ctx);
   return next();
 }
 
+function enableDisableToggle(ctx) {
+  if (ctx && ctx.toggles) {
+    if (ctx.query.toggles) {
+      const reqFeatureToggle = ctx.query.toggles;
+      const onlyFeatureToggleName = reqFeatureToggle.substr(
```

Comment:
You could use a regex replace here to be `toggle.replace(/^\!(.*)/, '$1')` which would give you the name for both `not` and `is`. 

[e.g](https://www.typescriptlang.org/play?#code/MYewdgzgLgBGKwLwwOQEIoAsCWYDmEKAsAFCiSwC2AhlMJjMilrgcWeNDDXZgEyNULNPkKlS5CCAA2AUwB0uAGYgAFPCjyATrIAO06sFmqA9AD0AOmlXyAVAEoTAGlQASAIwp79iZxkLlNR56bT0DI1NLaztHFxQPLx8OSH9FMBVVYP5Q-UNjcysbB2c3T28gA)